portfolio/src/main/webapp/comment.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Fetches the response of "/comment".
+ */
+function loadAndShowComments() {
+    let maxNumberOfComments = document.getElementById("quantity").value;
+    if (maxNumberOfComments.length == 0) {
+        maxNumberOfComments = 0;
+    }
+    let url = "/list-comment?quantity=" + maxNumberOfComments;
+    fetch(url, {method: "GET"}).then(response => response.json()).then((json) => {
+        const div = document.getElementById("comments");
```

Comment:
Maybe this part can be in showComments(comments)

Reply:
Which part?

Reply:
As discussed offline, i wrap whatever inside the lambda block that do the showing json logic into showComments(comments)